packages/pieces/community/text-helper/src/index.ts

```diff
@@ -6,14 +6,30 @@ import { htmlToMarkdown } from './lib/actions/html-to-markdown';
 import { markdownToHTML } from './lib/actions/markdown-to-html';
 import { replace } from './lib/actions/replace';
 import { split } from './lib/actions/split';
+import { stripHtmlContent } from './lib/actions/strip-html';
 
 export const textHelper = createPiece({
   displayName: 'Text Helper',
   description: 'Tools for text processing',
   auth: PieceAuth.None(),
   logoUrl: 'https://cdn.activepieces.com/pieces/text-helper.svg',
-  authors: ["joeworkman","kishanprmr","MoShizzle","AbdulTheActivePiecer","abuaboud"],
+  authors: [
+    'joeworkman',
+    'kishanprmr',
+    'MoShizzle',
+    'AbdulTheActivePiecer',
+    'abuaboud',
+    'AdamSelene'
```

Comment:
@AdamSelene, could you please add a comma (,) after your name? It will fix the lint issue.

Reply:
Done!